engine-vuex/src/store.ts

```diff
@@ -334,11 +335,11 @@ export class WWTEngineVuexModule extends VuexModule implements WWTEngineVuexStat
       this.isTourPlaying = false;
     }
 
-    const showWebGl2Warning = !wwt.si.isUsingWebGl2()
-      && (Date.now() - this.timeAtStartup) < 15000;
-    if (this.showWebGl2Warning != showWebGl2Warning) {
-      this.showWebGl2Warning = showWebGl2Warning;
-    }
+    // const showWebGl2Warning = !wwt.si.isUsingWebGl2()
+    //   && (Date.now() - this.timeAtStartup) < 15000;
+    // if (this.showWebGl2Warning != showWebGl2Warning) {
+    //   this.showWebGl2Warning = showWebGl2Warning;
+    // }
   }
```

Comment:
I commented this out because it was incessantly giving an error in the console on my machine. I assume that shouldn't be the case, though?

Reply:
Interesting... You don't happen to use Safari or some old version of a browser, do you?

The intention is to show a 15 second warning to users with browsers not supporting WebGL 2.

Reply:
Turns out this was all on my end - I was using a bit out-of-date version of the `scriptsharp` artifact . Using the newest version, I don't get this error at all.